internal/server/v1/propertyvalues/golden/bulk_property_values_in/containedIn1.json

```diff
@@ -52187,12 +52187,12 @@
           "provenanceId": "dc/91s0hm"
         },
         {
-          "name": "Chawanakee Unified School District",
+          "name": "Chawanakee Unified",
           "types": [
             "SchoolDistrict"
           ],
           "dcid": "geoId/sch0600116",
-          "provenanceId": "dc/sm3m2w3"
+          "provenanceId": "dc/91s0hm"
```

Comment:
School district changes are also expected because this is actually a newly added provenance